code/xcvm/cosmwasm/cw-substrate-cli/src/new_cmd.rs

```diff
@@ -0,0 +1,76 @@
+use crate::error::Error;
+use anyhow::anyhow;
+use clap::Args;
+use std::process::Command;
+
+const CARGO_GENERATE_COMMAND_NAME: &str = "generate";
+const CARGO_TEMPLATE_KEY_PROJECT_DESCRIPTION: &str = "project_description";
+const CARGO_TEMPLATE_KEY_PROJECT_AUTHORS: &str = "project_authors";
+const CARGO_TEMPLATE_GIT: &str = "https://github.com/ComposableFi/cw-template-project";
+const CARGO_GENERATE_HELP_MESSAGE: &str = r#"
+cargo-generate is not installed. Please install it by running `cargo install cargo-generate`.
+See more at: https://github.com/cargo-generate/cargo-generate
+"#;
+
+#[derive(Args, Debug)]
+/// Create a base CosmWasm project.
+pub struct NewCommand {
+	/// Project name
+	#[arg(short, long)]
+	name: Option<String>,
+
+	/// Project description
+	#[arg(short, long)]
+	description: Option<String>,
+
+	/// Project authors
+	#[arg(short, long)]
+	author: Option<String>,
```

Comment:
Can we default this to the git.username + git.email like cargo?

Reply:
actually `cargo generate` seems to have a default placeholder called `{{authors}}`, so we don't need this at all. cool